src/projects/projects.service.ts

```diff
@@ -1,26 +1,76 @@
 import { Injectable } from '@nestjs/common';
 import { CreateProjectDto } from './dto/create-project.dto';
-import { UpdateProjectDto } from './dto/update-project.dto';
+import { Model } from 'mongoose';
+import { InjectModel } from '@nestjs/mongoose';
+import { Projects } from './interfaces/projects.interface';
 
 @Injectable()
 export class ProjectsService {
-  create(createProjectDto: CreateProjectDto) {
-    return 'This action adds a new project';
-  }
+  constructor(@InjectModel('Projects') private projectsModel: Model<Projects>) {}
 
-  findAll() {
-    return `This action returns all projects`;
+  async create(userId: number, createProjectDto: CreateProjectDto): Promise<Projects> {
```

Comment:
Is this used to create a user or project? Seems that this is violating the SRP principle because it is doing both. Dto or newly created object also is not validated so a project without a name could be added to the database if I am not wrong which can lead to an unwanted situations.